db/queries.py

```diff
@@ -1,46 +1,85 @@
 from db.models import Message, User, Chat
-from django.db.models import Q, Count, F
+from django.db.models import Q, Count
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(
+        Message.objects.filter(text__icontains=word)
+    )
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return list(
+        Chat.objects.filter(title__startswith="Untitled")
+    )
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
```

Comment:
incorrect return value typehint

Reply:
`str` there is noted by default. Acc to tests in fact task ask me list(tuple(str))). So i've just changed a hint